pages/help_dialog.js

```diff
@@ -124,7 +124,17 @@ const HelpDialog = {
             });
           }
 
-          $$(descriptionElement, ".vimiumHelpDescription").textContent = command.description;
+          const MAX_LENGTH = 50;
+          const desiredOptionsLength = Math.max(0, MAX_LENGTH - command.description.length - 3);
+          const optionsTruncated = command.options.substring(0, desiredOptionsLength);
+          let ellipis = "";
+          if ((command.description.length + command.options.length) > MAX_LENGTH) {
+            ellipis = "...";
+            $$(descriptionElement, ".vimiumHelpDescription").title = command.options;
+          }
+          const optionsString = command.options ? ` (${optionsTruncated}${ellipis})` : "";
+          const fullDescription = `${command.description}${optionsString}`;
```

Comment:
This confused me because it looks like here we're setting the textContent to be a truncated options string, but it may have not been truncated.

I think this can be made clearer by just mutating the display string we're going to show. Roughly:

    let optionDisplayString = command.options.substring;

    if (we need to truncate) {
      optionDisplayString = command.options.substring(0, desiredOptionsLength) + "...";
      $$(descriptionElement, ".vimiumHelpDescription").title = command.options;
    }

    const optionsString = command.options ? ` (${optionDisplayString})` : "";
    const fullDescription = `${command.description}${optionsString}`;

Reply:
I agree, without thoroughly reading the code it does look like the string is always being truncated. I'll change it to be more intuitive.